devtools/ci/ci_main.sh

```diff
@@ -59,7 +59,7 @@ case $GITHUB_WORKFLOW in
     make check-whitespaces
     make check-dirty-rpc-doc
     make check-dirty-hashes-toml
-    devtools/ci/check-cyclic-dependencies.py
+    devtools/ci/check-cyclic-dependencies.py --dev
```

Comment:
we didn't check the exit code of this script, should we fix it?

Reply:
Yes it should be fixed